handlers/handlers.go

```diff
@@ -39,29 +40,10 @@ func setStatusCode(req *http.Request, w http.ResponseWriter, err error) {
 	w.WriteHeader(status)
 }
 
-// getCookiePreferencePage talks to the renderer to get the cookie preference page
-func getCookiePreferencePage(w http.ResponseWriter, req *http.Request, rendC RenderClient, cp cookies.Policy, isUpdated bool, lang string) error {
-	var err error
-	ctx := req.Context()
-	m := mapper.CreateCookieSettingPage(cp, isUpdated, lang)
-	b, err := json.Marshal(m)
-	if err != nil {
-		log.Event(ctx, "unable to marshal cookie preferences", log.Error(err))
-		setStatusCode(req, w, err)
-		return err
-	}
-
-	templateHTML, err := rendC.Do("cookies-preferences", b)
-	if err != nil {
-		log.Event(ctx, "getting template from renderer cookies-preferences failed", log.Error(err))
-		setStatusCode(req, w, err)
-		return err
-	}
-	if _, err := w.Write(templateHTML); err != nil {
-		log.Event(ctx, "error on write of cookie template", log.Error(err))
-		setStatusCode(req, w, err)
-	}
-	return err
+// getCookiePreferencePage builds the cookie preference page using the rendering library
+func getCookiePreferencePage(cfg *config.Config, w http.ResponseWriter, req *http.Request, rendC RenderClient, cp cookies.Policy, isUpdated bool, lang string) {
```

Comment:
A lot of the error handling that was being done previously may no longer be needed as we no longer need to marshal any of the data or send it via an additional request